swap_meet/electronics.py

```diff
@@ -1,2 +1,15 @@
-class Electronics:
-    pass
+from uuid import uuid4
+from swap_meet.item import Item
+
+class Electronics(Item):
+    def __init__(self, id = None, type = "Unknown", condition = 0):
+        super().__init__(id = id, condition = condition)
+        self.type = type
+        
+    def get_category(self):
+        return "Electronics"
+    
+    def __str__(self):
+        return f"An object of type Electronics with id {self.id}. This is a {self.type} device."
```

Comment:
Leverage parent class's method so you can DRY up your code. 

```suggestion
        return f"{super().__str__()}. This is a {self.type} device."
```